modules/replicator/src/main/java/org/apache/ignite/internal/replicator/ReplicaManager.java

```diff
@@ -185,7 +185,20 @@ public void onReceived(NetworkMessage message, NetworkAddress senderAddr, @Nulla
                     Replica replica = replicas.get(request.groupId());
 
                     if (replica == null) {
-                        //TODO:IGNITE-17514 Send an exceptional response to the client side when the replica is absent.
+                        var traceId = UUID.randomUUID();
+
+                        clusterNetSvc.messagingService().respond(
+                                senderAddr,
+                                REPLICA_MESSAGES_FACTORY
+                                        .errorReplicaResponse()
+                                        .errorMessage(
+                                                IgniteStringFormatter.format("Replica is not ready "
+                                                        + "[replicaGrpId={}, nodeId={}]",
+                                                        request.groupId(), clusterNetSvc.topologyService().localMember().id()))
```

Comment:
Let's just use localMember() instead of localMember().id(). In many cases it's useful to see consistentId, volatileId, address, etc. Please also replace nodeId={} with node={} and replicaGrpId with replicationGroupId.